src/App.jsx

```diff
@@ -14,33 +15,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const ALPHABETICAL_ORDER = 'alphabet';
+const LENGTH_ORDER = 'length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortGoods, reversed }) {
```

Comment:
You can move this function outside of the file, then import and use it inside the component.